### PR TITLE
Peer review for BXMSDOC-2776-master Adding related links and minor content tweaks for…

### PR DESCRIPTION
**Pull request by michelehaglund**
Rendered output: http://file.rdu.redhat.com/~mhaglund/BXMSDOC-2776/

Original Jira: https://issues.jboss.org/browse/BXMSDOC-2776

### CODE REVIEW

File: docs/product-getting-started-guide/src/main/asciidoc/data-model-proc.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [id='_defining_a_data_model']
 = Viewing the *Mortgage Process* data objects
 
-The goal of this chapter is to introduce you to the *Mortgage Process* data objects.
+The goal of this chapter is to introduce you to the *Mortgage Process* data objects.  This tutorial utilizes the predefined data objects in the *Mortgage Process* sample project to save time in completing the mortgage application scenario.
```
Comment (mramendi) [inline docs/product-getting-started-guide/src/main/asciidoc/data-model-proc.adoc]: This got a bit confusing. It is in intro to "3.2" but speaks of "this chapter" (chapter is the entire "3") and then even "this tutorial".

File: docs/product-getting-started-guide/src/main/asciidoc/decision-tables-proc.adoc [hunk 2]
```diff
@@ -6,6 +6,9 @@ Decision tables are a way to generate rules driven from the data entered into a
 .Prerequisites
 The business rules have been defined. For more information, see <<_defining_business_rules>>.
 
+.Related information
+For more information on decision tables, see {URL_GUIDED_DECISION_TABLES}[_{GUIDED_DECISION_TABLES}_].
```
Comment (mramendi) [inline docs/product-getting-started-guide/src/main/asciidoc/decision-tables-proc.adoc]: Do we use "information on" or "information about"? I remember being corrected from "on" to "about" back in my IBM days. But the promised copy of the IBM Style Guide never reached me.

Comment (mramendi) [reply docs/product-getting-started-guide/src/main/asciidoc/decision-tables-proc.adoc]: (same applies to other instances of "information on")

File: docs/product-user-guide/src/main/asciidoc/data-objects-create-proc.adoc [hunk 3]
```diff
@@ -3,11 +3,12 @@
 //product-assembly_guided-rules/../main.adoc
 //product-assembly_guided-rule-templates/../main.adoc
 //product-user-guide/../chap-data-models
+//product-assembly-business-processes/../main.adoc
```
Comment (mramendi) [inline docs/product-user-guide/src/main/asciidoc/data-objects-create-proc.adoc]: Not sure what this change is about - it does not seem to affect doc

File: docs/product-getting-started-guide/src/main/asciidoc/data-model-proc.adoc (hunk 1 above)
Comment (michelehaglund) [reply docs/product-getting-started-guide/src/main/asciidoc/data-model-proc.adoc]: Agreed and simplified. Good catch!

File: docs/product-getting-started-guide/src/main/asciidoc/decision-tables-proc.adoc (hunk 2 above)
Comment (michelehaglund) [reply docs/product-getting-started-guide/src/main/asciidoc/decision-tables-proc.adoc]: Great catch. Just checked the Style guide and it is "Information about". Never "information on". Fixing throughout.

File: docs/product-user-guide/src/main/asciidoc/data-objects-create-proc.adoc (hunk 3 above)
Comment (michelehaglund) [reply docs/product-user-guide/src/main/asciidoc/data-objects-create-proc.adoc]: Me either. Just following what was already there :)

**Pull request merged**